regexredux/regexredux.jl

```diff
@@ -7,7 +7,7 @@
 
```

Comment:
Should the changes to this file be cherry-picked out of this branch as an improvement to the existing Julia program?